Solutions/Cisco ISE/Parsers/CiscoISEEvent.yaml

```diff
@@ -11,29 +11,61 @@ FunctionQuery: |
     | project-rename DvcIpAddr = ['Device IP Address']
     				,DvcHostname = NetworkDeviceName
     				,DstIpAddr = DestinationIPAddress
     				,DstPortNumber = DestinationPort
     				,DstUserName = UserName
+                    ,NetworkDeviceProfile = ['Network Device Profile']
+                    ,AcctDelayTime = ['Acct-Delay-Time']
+                    ,AllowEasyWiredSession= allowEasyWiredSession
+                    ,AcctStatusType = ['Acct-Status-Type']
+                    ,AcctOutputOctets = ['Acct-Output-Octets']
+                    ,AcctInputOctets = ['Acct-Input-Octets']
+                    ,SubjectCommonName = ['Subject - Common Name']
+                    ,TLSCipher= TLSCipher
+                    ,IsMachineAuthentication= IsMachineAuthentication
+                    ,IsMachineIdentity = IsMachineIdentity
+                    ,ADHostQualifiedName = ['AD-Host-Qualified-Name']
+                    ,TLSVersion= TLSVersion
+                    ,DTLSSupport= DTLSSupport
+                    ,Port = Port
```

Comment:
why renaming to same name?

<img width="530" height="213" alt="image" src="https://github.com/user-attachments/assets/f566a9f7-7ea5-4ad6-a035-045b1fbe6b24" />

Reply:
<img width="549" height="178" alt="image" src="https://github.com/user-attachments/assets/fbe6f3b5-5261-4a87-8265-8400c0c1e87a" />

Reply:
<img width="563" height="155" alt="image" src="https://github.com/user-attachments/assets/260560ac-b456-4dcc-81d8-57c402020ec8" />